master/buildbot/data/base.py

```diff
@@ -88,6 +88,9 @@ def control(self, action, args, kwargs):
     def startConsuming(self, callback, options, kwargs):
         raise NotImplementedError
 
+    def __repr__(self):
+        return "endpoint for" + self.pathPatterns
```

Comment:
I don't have a preference of one approach over the other, but yes -- the missing space needs to be added.  With that fixed, I think this is ready to merge.